src/cf-layout/src/molecules/heroes.less

```diff
@@ -6,7 +6,7 @@
         padding-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
 
         .respond-to-min( @bp-sm-min, {
-            // IE specific fix
```

Comment:
I especially prefer the IE specific hints, cause otherwise I'd expect I could optimize the code by removing them when working on this in a modern browser in the future.